newIDE/app/src/GameDashboard/LeaderboardAdmin/index.js

```diff
@@ -0,0 +1,835 @@
+// @flow
+import React from 'react';
```

Comment:
Prefer `import * as React from 'react'` to benefit from `React.Node` types

Reply:
Oh yes thanks!